### PR TITLE
Google Internship Virtual reality 

### PR DESCRIPTION
**Pull request by Muneeb1997**
Any one plz guide me how i can apply.

### CODE REVIEW

File: docs/tsc/meetings/2019-10-21.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  * [ ] Mark Boorer (_TSC_) - Industrial Light & Magic
+  * [ ] Sean Cooper (_TSC_) - DNEG
+  * [X] Michael Dolan (_TSC Chair_) - Sony Pictures Imageworks
+  * [ ] Larry Gritz (_TSC_) - Sony Pictures Imageworks
+  * [X] Patrick Hodoul (_TSC_) - Autodesk
+  * [X] John Mertic - Academy Software Foundation / Linux Foundation
+  * [X] Carl Rand (_TSC_) - Foundry
+  * [X] Doug Walker (_TSC Chief Architect_) - Autodesk
+  * [X] Kevin Wheatley (_TSC_) - Framestore
+  * [X] Bernard Lefebvre - Autodesk
```
Comment (Muneeb1997) [inline docs/tsc/meetings/2019-10-21.md]: git commit --amend --signoff

**Comment by hodoulp** [comment]
Hi @Muneeb1997 

If you want to apply to the GSoC, you should go to the [ocio-dev](https://lists.aswf.io/g/ocio-dev/topics) group for questions and close this pull request. @michdolan Could you add him to slack gsoc?

The OpenColorIO (aka OCIO) project is part of the [Academy Software Foundation](https://www.aswf.io/) (aka ASWF) which requests for at least one valid pull request for all students who postulate to the GSoC. 

For students, the [GSoC](https://summerofcode.withgoogle.com/) apply period ends 31st of march.

**Comment by hodoulp** [comment]
Here is the list of [proposed projects](https://github.com/AcademySoftwareFoundation/tac/tree/master/gsoc) for OCIO. But you can still propose something else following your interests.

**Comment by Muneeb1997** [comment]
I am still confuse about it...

**Comment by hodoulp** [comment]
@Muneeb1997 To apply to GSoC, you need to make a written proposal (i.e. a project representing roughly 3-month work) to be posted on the GSoC web site. To help students, we listed few projects to give some hints. But we've added an extra condition about doing a successful pull request.

Remember that the deadline is the 31st of march i.e. today.

**Comment by Muneeb1997** [comment]
Is it also need gives a demo on Arduino code too?   Can I send you my
proposal for review?

Thanks.

**Comment by Muneeb1997** [comment]
Here is my Proposal:

https://drive.google.com/file/d/1bAbVqa0O0IfdWiVUiR9_aOwEbEco1R_-/view?usp=sharing

Please have a review and let me know what you think?

Thanks.

**Comment by hodoulp** [comment]
@Muneeb1997 The proposal must be related to the OpenColorIO project. Based on your proposal you need to find an Open Source project related to VR and/or Game.

**Pull request closed without merging**